src/streaming/controllers/AbrController.js

```diff
@@ -461,6 +466,22 @@ function AbrController() {
         return newIdx;
     }
 
+    /**
+     * Returns the maximum possible index from CMSD model
+     * @param type
+     * @param streamId
+     * @return {number|*}
+     */
+    function _checkCmsdMaxBitrate(idx, type, streamId) {
+        const maxCmsdBitrate = cmsdModel.getMaxBitrate(type);
+        if (maxCmsdBitrate < 0) {
+            return idx;
+        }
+        logger.info('Stream ID: ' + streamId + ' [' + type + '] Apply max bit rate from CMSD: ' + maxCmsdBitrate);
+        const maxIdx = getQualityForBitrate(streamProcessorDict[streamId][type].getMediaInfo(), maxCmsdBitrate, streamId);
+        return Math.min(idx, maxIdx);
```

Comment:
At this point we are not exactly following the spec that states in 8.3 that we should switch to a combination of video and audio for which the aggregate bitrate is less or equal than `mb`